### ISSUE TITLE
sticky Context where it shouldn't

### ISSUE THREAD

**Issue by dragoncoder047** [technoblogy/ulisp-esp#61]
```
9043> (progn (terpri) (foo))

Error: 'terpri' undefined: foo
9043>
```
Fix (in `eval()`):

```diff
   if (symbolp(form)) {
        symbol_t name = form->name;
        object* pair = value(name, env);
        if (pair != NULL) return cdr(pair);
        pair = value(name, GlobalEnv);
        if (pair != NULL) return cdr(pair);
        else if (builtinp(name)) return form;
+       Context = NIL;
        error(PSTR("undefined"), form);
    }

**Comment by technoblogy** [comment]
Good catch - thanks. Will be fixed next release.

**Comment by technoblogy** [comment]
Fixed in 4.4b - thanks!
